bls/bignum_pure.go

```diff
@@ -107,8 +108,12 @@ func PowModFr(dst *Fr, a, b *Fr) {
 	(*big.Int)(dst).Exp((*big.Int)(a), (*big.Int)(b), &_modulus)
 }
 
-func InvModFr(dst *Fr, v *Fr) {
+func InvModFr(dst *Fr, v *Fr) error {
```

Comment:
The idea is that this function signature matches `InvModFr` in:
- Herumi BLS: https://github.com/protolambda/go-kzg/blob/c91cee5e954e1506cc940d404a7918e17b258749/bls/bignum_hbls.go#L104
- Holiman uint256: https://github.com/protolambda/go-kzg/blob/c91cee5e954e1506cc940d404a7918e17b258749/bls/bignum_hol256.go#L118
- Kilic BLS (default): https://github.com/protolambda/go-kzg/blob/c91cee5e954e1506cc940d404a7918e17b258749/bls/bignum_kilic.go#L113

With build tags the performance of different implementations of field-element operations can be compared in benchmarks etc. since a big part of this repository is FFT related code for data-availability-sampling.

And since native divide by zero behaves the same with a panic, and this error return is a breaking API change, I'm a bit hesitant to merge this. What do you use go-kzg for, and what is the motivation for this PR?

Reply:
I'm just exploring your repo for the sake of learning. This PR is for error handling - might be helpful if used at scale